fe/fe-core/src/main/java/org/apache/doris/catalog/external/EsExternalDatabase.java

```diff
@@ -135,12 +145,14 @@ public Set<String> getTableNamesWithLock() {
     @Override
     public List<EsExternalTable> getTables() {
         makeSureInitialized();
+        MasterCatalogExecutor.waitJournalReplay(replayToJournalId);
```

Comment:
Why not move `MasterCatalogExecutor.waitJournalReplay(replayToJournalId);` into `makeSureInitialized()`?